tmux-setup/index.html

```
@@ -0,0 +1,107 @@
+        <a href="/configuration/">Configuration</a>
+        <a href="/tui-picker/">TUI picker</a>
+        <a href="/about/">About</a>
+        <a class="external" href="https://github.com/alchemmist/lazy-tmux">GitHub ↗</a>
+      </nav>
+    </aside>
+    <main class="content">
+      <div class="page">
+        <section class="doc-section">
+<h1>Quick tmux setup</h1>
+<p>Keep your tmux config clean with an include file:</p>
+        <pre><code>lazy-tmux setup >> ~/.tmux.conf</code></pre>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)